rasa_nlu/data_router.py

```diff
@@ -35,6 +54,15 @@ def __init__(self, config, component_builder):
         self.emulator = self.__create_emulator()
         self.component_builder = component_builder if component_builder else ComponentBuilder(use_cache=True)
         self.model_store = self.__create_model_store()
+        self.pool = ProcessPool(config['max_training_processes'] or 1)
```

Comment:
No, it's called belt and suspenders engineering. I'll check for `"max_training_processes"<=0`